pkg/storage/postgresql/issues.go

```diff
@@ -150,6 +152,8 @@ func (db DB) IssuesSummary(filter storage.Filter) ([]model.IssueSummary, storage
 				{{ if and (eq .Status "FIXED") (or .MinDate .MaxDate) }}
```

Comment:
Same comment as in `ListFindings` method, I think we are missing the explicit ignore of FALSE POSITIVE findings from the "else branch" when a `AtDate` filter is used without a "status" filter.

Reply:
If you don't mind, I will resolve this comment when we agree on what to do with the `ListFindings` method.